plugins/bitrise-ci/src/components/BitriseDownloadArtifactComponent/BitriseDownloadArtifactComponent.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import CloudDownload from '@material-ui/icons/CloudDownload';
+import { CircularProgress, IconButton, Tooltip } from '@material-ui/core';
+import LinkIcon from '@material-ui/icons/Link';
+import { useBitriseArtifactDetails } from '../useBitriseArtifactDetails';
+import { Alert } from '@material-ui/lab';
+
+type BitriseDownloadArtifactComponentProps = {
+  appSlug: string;
+  buildSlug: string;
+  artifactSlug: string;
+};
+
+export const BitriseDownloadArtifactComponent = ({
+  appSlug,
+  buildSlug,
+  artifactSlug,
+}: BitriseDownloadArtifactComponentProps) => {
+  const { value, loading, error } = useBitriseArtifactDetails(
+    appSlug,
+    buildSlug,
+    artifactSlug,
+  );
+
+  if (loading) {
+    return <CircularProgress />;
+  }
+
+  if (error) {
+    return <Alert severity="error">{error.message}</Alert>;
```

Comment:
Some of these could be refactored to use:

https://backstage.io/storybook/?path=/story/feedback-warning-panel--default

Reply:
@adamdmharvey I had a look at the warning panel but that does not provide different "shades" or warnings/errors, or? 
The use case here is that there was an artefact created but its expired and therefore not available for download anymore - where the hard `error` is shown when the artefact is not present although it should.